recipes/wgbs_tools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "wgbs_tools" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('wgbs_tools',min_pin='x.x', max_pin='x.x') }}
+#  noarch: python
+
+source:
+  path: ./
+  sha256: unused
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    - python >=3.6
+    - numpy 
+    - pandas
+    - gxx
+ 
+host:
```

Comment:
This section needs to be indented to be under requirements

Reply:
Thank you! The error was really changed to 
`ModuleNotFoundError: No module named 'pandas'`

Reply:
Thank you **very** much!